web/src/utils/getSnapPoints.ts

```diff
@@ -1,9 +1,16 @@
+import dimensions from '../constants/dimensions'
+
 const min = 0.05
 const mid = 0.35
-const max = 0.65
+const mapIconsHeight = 60
+
+// Calculates detail snapPoint by reducing height with navigation and iconHeight on the map
+const getDetailSnapPoint = (maxHeight: number) =>
+  maxHeight - dimensions.headerHeightLarge - dimensions.navigationMenuHeight - mapIconsHeight
+
 export const getSnapPoints = (maxHeight: number): number[] => [
   maxHeight * min,
   maxHeight * mid,
-  maxHeight * max,
+  getDetailSnapPoint(maxHeight),
```

Comment:
I think you could just inline this with a comment, no need to extract to a function. But fine like it is :)